fusequery/query/src/sql/plan_parser.rs

```diff
@@ -208,6 +227,86 @@ impl PlanParser {
         }))
     }
 
+    fn insert_to_plan(
```

Comment:
Hello @hulunbier 
Here we need some tests for the insert plan.
There are already some cases in:
https://github.com/datafuselabs/datafuse/blob/748fe40d3bbd5bbf034254dbb1d5f077d2ffbbec/fusequery/query/src/sql/plan_parser_test.rs